_docs/14-animation.md

```diff
@@ -72,6 +74,190 @@ Override this method to specify how the top-level bounds of a component should a
 
 ## Notes
 
-If you implement either method, your component must have a [scope](scopes.html).
+**If you implement either method, your component must have a [scope](scopes.html).**
 
 If updating your component's [state](state.html) changes it's bounds, both `boundsAnimationFromPreviousComponent` and `animationsFromPreviousComponent` will be called. 
+
+## In-depth
+
+### UIKit Animations
+
+Engineers on iOS are used to the simple UIView implicit animations. Here’s what they look like:
+
+{% highlight objc %}
+UIView *view = ...;
+[UIView animateWithDuration:0.3 animations:^{
+	view.frame = CGRectMake(50, 50, view.frame.size.width, view.frame.size.height);
+}];
+{% endhighlight %}
+
+With this little bit of code, we’ve taken a UIView, and changed its frame’s origin to be (50, 50).  This is what’s called an implicit animation, since what we’ve done here is just alter a  property within a block. UIKit records mutations  to its model data within this block, and forms a transaction that it bundles into an animation for the render server lasting 300 ms. from the view’s current position to its new position.
+
+There are all kinds of properties I can animate with this API: alpha, scale, rotation, Z position. Implicit animations can control their durations, easing curves, and can specify that the animated property should begin at its current display or model properties.
+
+This API is used for the majority of animations that you see in apps on iOS.
+
+#### CoreAnimation
+
+CoreAnimation is the underlying rendering engine that is used on iOS and most of Mac OS to render contents on the screen using OpenGL. It operates in a different process from our app. It has its own representations of all the layers on the screen, and their current properties, as displayed to the user. Mutations to the views and layers inside our app get packaged into a transaction which are transmitted to the render server through a special-purpose XPC bridge.
```

Comment:
Modern CoreAnimation uses Metal to render contents to the screen. Also, not sure if CoreAnimation uses XPC for its IPC.